### DISCUSSION TITLE
Retrieving SQL credentials in devOps pipeline

### DISCUSSION THREAD

**Discussion by pascalm23** [Azure-Player/azure.synapse.tools#25] [Q&A] [answered]
Hi,

I'm using the ADF version of that script which works great. I'm trying to do the same for Synapse, but noticed that credentials are required here:
-SqlAdministratorLoginCredential $cred

I'm using a pipeline devOps (powershell task in a yaml file) and was wondering if someone could give an example on how to initialize $cred variable.

Thanks,
Pascal.

**Comment by pascalm23** [comment]
script being: Publish-SynapseFromJson.ps1

**Comment by NowinskiK** [comment, marked as answer]
Check this out:
https://stackoverflow.com/questions/73648383/azure-pipelines-setting-and-passing-the-password-in-powershell-script
